### PR TITLE
0.3.0 - use pip, change dependencies

### PR DESCRIPTION
**Pull request by jsignell**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jsignell** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+{% set version = "0.3.0" %}
 
 package:
    name: intake-xarray
    version: {{ version }}
 
 source:
-  url: https://github.com/ContinuumIO/intake-xarray/archive/{{ version }}.tar.gz
-  sha256: d64c61040ab02b19e3c1c2e329cd950074a2624091d21945d672612fa090b438
+  url: https://pypi.io/packages/source/i/intake-xarray/intake-xarray-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml:8]: We probably cannot run the full test suite with the PyPI source. (I'm fine dropping the tests BTW.)

Comment (jsignell) [reply recipe/meta.yaml:8]: Yeah I don't know why they are failing though

Comment (ocefpaf) [reply recipe/meta.yaml:8]: My guess is missing files that are not part of the source distribution.

Comment (jsignell) [reply recipe/meta.yaml:8]: yeah that's what @martindurant said too

**Comment by ocefpaf** [comment]
Thanks @jsignell!

**Pull request merged**